policyengine_us/parameters/gov/states/nm/tax/income/exemptions/armed_forces_retirement_pay/cap.yaml

```diff
@@ -0,0 +1,20 @@
+description: New Mexico caps the armed forces retirement pay exemptions at this amount.  
+values:
+  2022-01-01: 10_000
+  2023-01-01: 20_000
+  2024-01-01: 30_000
+  # expires after 2026
+metadata:
+  unit: currency-USD
+  period: year
+  label: New Mexico armed forces retirement pay exemption cap
+  reference:
+    - title: N.M. Stat. § 7-2-5.13 Exemption; armed forces retirement pay
+      href: https://casetext.com/statute/new-mexico-statutes-1978/chapter-7-taxation/article-2-income-tax-general-provisions/section-7-2-513-effective-until-112025-exemption-armed-forces-retirement-pay
+    - title: 2022 PIT-ADJ SCHEDULE OF ADDITIONS, DEDUCTIONS, AND EXEMPTIONS Line 23
```

Comment:
please remove all references that dont show the exact parameter amount - need to find the 2021 , 2022, 2023 tax forms that do

Reply:
or 2022 and 2023 if not implemented in 2021

Reply:
tax forms don't show the amount, only the instructions and legal code has the amount